controller/space_test.go

```diff
@@ -402,6 +403,25 @@ func (rest *TestSpaceREST) TestListSpacesOK() {
 	}
 }
 
+func (rest *TestSpaceREST) TestListSpacesOKEmptyResult() {
+	// delete all existing spaces
+	svc, ctrl := rest.SecuredController(testsupport.TestIdentity)
+	_, existingSpaces := test.ListSpaceOK(rest.T(), svc.Context, svc, ctrl, nil, nil, nil, nil)
+	spaceRepository := space.NewRepository(rest.DB)
+	for _, existingSpace := range existingSpaces.Data {
+		rest.T().Log("Deleting existing space with ID=", existingSpace.ID.String())
+		spaceRepository.Delete(svc.Context, *existingSpace.ID)
+	}
+	// when
+	res, emptyList := test.ListSpaceOK(rest.T(), svc.Context, svc, ctrl, nil, nil, nil, nil)
+	// then
+	require.NotNil(rest.T(), *emptyList)
+	require.Empty(rest.T(), emptyList.Data)
+	var defaultTime time.Time
+	assert.NotEqual(rest.T(), defaultTime.Format(time.RFC1123), res.Header().Get(app.LastModified))
```

Comment:
Use http.TimeFormat instead..  http.ParseDate doesn't understand RFC1123, HTTP Dates are hardcoded to GMT